app.py

```diff
@@ -9,7 +9,7 @@
 app = Flask(__name__)
 app.config.from_object(Config)
 
-db = Database(app, "cs361_edmondem", '3152')
+db = Database(app, "cs361_alberjes", '3526')
```

Comment:
Gosh we are all so bad about this :laughing: guess it doesn't really matter at this point.

Reply:
Yeah, I'm not too worried about it now. Haha